client/flutter/lib/pages/symptom_checker/question_pages/short_list_question_view.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_html/flutter_html.dart';
+import 'package:flutter_svg/svg.dart';
```

Comment:
super-nit / not a blocker: other files import from `flutter_svg/flutter_svg.dart` which [looks like](https://github.com/dnfield/flutter_svg/blob/master/lib/flutter_svg.dart) it just exports the `svg.dart` file, so you could import from the same:

```suggestion
import 'package:flutter_svg/flutter_svg.dart';
```

Reply:
@hspinks don't think we should do it in this PR, due to:

<img width="469" alt="Screen Shot 2020-05-05 at 11 04 08 PM" src="https://user-images.githubusercontent.com/11729521/81142536-c4b4cf80-8f24-11ea-8e87-b007ab9a053b.png">